src/stdlib_sorting_sort_index.fypp

```diff
@@ -121,7 +125,12 @@ contains
                 call merge_sort( array, index, work, ibuf )
             end if
         else
+            #:if t1[0] == "c"
```

Comment:
As noted previously -- consider comparing with "char" instead of "c"

Reply:
Done